### ISSUE TITLE
Kernel dead when applied FAMD to large dataset

### ISSUE THREAD

**Issue by dlu19** [MaxHalford/prince#44]
I have a dataset X with mixed data. The shape of X is (110000, 24).
I applied the following command to this dataset and the kernel is dead. Any suggestions for this? Thanks! 

famd = prince.FAMD(
n_components=24,
n_iter=10,
copy=True,
engine='auto',
random_state=42
)
famd = famd.fit(X)

**Comment by MaxHalford** [comment]
Hey, do you happen to know what version of `prince` are you using?

**Comment by MaxHalford** [comment]
Also, can you give a bit more description about the columns? Do they contain contain categoricals? If so how many? It would be even better if you could provide the dataset (I'm aware this isn't always possible).

**Comment by dlu19** [comment]
> Hey, do you happen to know what version of `prince` are you using?

Hi Max, 

Thank you for your response. The version of prince I'm using is 0.4.9.
The shape of my dataset (X) is (102377, 24), which means that 102377 observations and 24 variables (features). Among those 24 features, there are 

- 21 continuous variables, and I've already standardized it with zero mean and 1 standard error. 

- 3 categorical variables, one with 4 categories, one with 3 categories, one with 2 categories.

When I applied the example code, the Jupyter kernel is dead:
famd = prince.FAMD(
n_components=24,
n_iter=10,
copy=True,
engine='auto',
random_state=42
)
famd = famd.fit(X)

Sorry I can't provide the detail dataset. Hope this description is helpful! Thank you so much!
